desktop/components/marketing_signup_modal/index.coffee

```diff
@@ -54,7 +54,8 @@ module.exports = class MarketingSignupModal extends Backbone.View
 
     # This is just for the Seattle Art Fair campaign and should be removed after
     # the fair closes.
-    slug = 'ca3' if !slug and sd.CURRENT_PATH is '/seattle-art-fair-2017'
```

Comment:
@craigspaeth so that might be something like `sd.FAIR_URL` or maybe `sd.TARGET_CAMPAIGN_URL` which would get set in the environment variables. What might that value be if we aren't currently running a campaign? An empty string?